src/new-client/src/Web/serviceWorkerRegistration.ts

```diff
@@ -28,7 +28,10 @@ type Config = {
 export function register(config?: Config) {
   if ("serviceWorker" in navigator) {
     window.addEventListener("load", () => {
-      const swUrl = `${window.location.origin}/sw.js`
+      // This url currently breaks when running against the env gateway as it misses the correct pathname
```

Comment:
this comment out of date now, I guess - this env deployment PWA works:
https://web.env.reactivetrader.com/branch/fix_manifest-file-for-pwa

Reply:
changed thanks